lab01/riccardo_rende.txt

```diff
@@ -0,0 +1 @@
+I started watching movies.
```

Comment:
Please, sign this below the phrase, so we know who you are when we merge all files...